app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def index
```

Comment:
Those blank methods again though! Talk to your team about why this isn't a great habit to get into and if they've forgotten let me know and I'll review it again 👍